karma-build.conf.js

```diff
@@ -9,6 +9,7 @@
 module.exports = function(config) {
   require('./karma-base.conf.js')(config);
   config.files.push('node_modules/core-js-bundle/index.js');
+  config.files.push('build/test/browser-env-setup.js');
```

Comment:
this seems unrelated. Can you explain the reason for this?

Reply:
In order to test the changes and protect against regressions, the tests are run with the symbol prefix changed to` __zone_symbol_test__`. This adds the environment setup that sets the `__Zone_symbol_prefix` optin.